src/static/pycontw-2020/styles/page.scss

```diff
@@ -381,7 +381,16 @@ main {
     }
 }
 
+// customize privacy_policy and guidelines page ol
+ol.custom_ol {
+margin-top: 20px;
```

Comment:
I didn't make myself clear. I meant we can have spacing *between each item* in the ordered list, which slightly improves the readability.

Reply:
okay , I got it .